packages/astro/test/get-static-paths-pages.nodetest.js

```diff
@@ -16,12 +17,12 @@ describe('getStaticPaths with trailingSlash: ignore', () => {
 	it('includes index page', async () => {
 		let html = await fixture.readFile('/index.html');
 		let $ = cheerio.load(html);
-		expect($('h1').text()).to.equal('Page 1');
+		assert.strictEqual($('h1').text(), 'Page 1');
```

Comment:
We do use `assert.equal` in all tests, that's because we already import `assert` from `node:assert/strict`. This means `assert.equal` is `assert.strictEqual`.

Would you mind changing that?

Reply:
Yes. If there are other instances of `strictEqual`, we can do that in another PR